<?xml version="1.0" encoding="UTF-8"?><testrun name="LibraryTest (1)" project="DummyTesting" tests="10" started="10" failures="1" errors="0" ignored="0">
  <testsuite name="DummyTesting.LibraryTest" time="0.01">
    <testcase name="givenEmailId_WhenNotProper_ShouldReturnFalse" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenFirstName_WhenProper_ShouldReturnTrue" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenPhoneNumber_WhenNotProper_ShouldReturnFalse" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenEmailId_WhenProper_ShouldReturnTrue" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenLastName_WhenNotProper_ShouldReturnFalse" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenFirstName_WhenNotProper_ShouldReturnFalse" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenLastName_WhenProper_ShouldReturnTrue" classname="DummyTesting.LibraryTest" time="0.001"/>
    <testcase name="givenPassword_WhenNotProper_ShouldReturnFalse" classname="DummyTesting.LibraryTest" time="0.0"/>
    <testcase name="givenPassword_WhenProper_ShouldReturnTrue" classname="DummyTesting.LibraryTest" time="0.009">
      <failure>java.lang.AssertionError&#13;
	at org.junit.Assert.fail(Assert.java:86)&#13;
	at org.junit.Assert.assertTrue(Assert.java:41)&#13;
	at org.junit.Assert.assertTrue(Assert.java:52)&#13;
	at DummyTesting.LibraryTest.givenPassword_WhenProper_ShouldReturnTrue(LibraryTest.java:60)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="givenPhoneNumber_WhenProper_ShouldReturnTrue" classname="DummyTesting.LibraryTest" time="0.0"/>
  </testsuite>
</testrun>
